oidc-provider/simple_op/src/provider/server/static/jwks.json

```diff
@@ -1 +1 @@
\ No newline at end of file
+{"keys": [{"use": "enc", "n": "3c2XFHeXcUuJq-QAgC5dFfDM4zLt47E9GKGzH6hTed88XmbJvuAaaRENcv99-9EUINUvcT0xk-D8nK-HCtUelzbRgQG1TkkMy2hKiFhW6EmTptyGCSQkdxP-skVavZ-1B7I0L3GJo9FmsimKsrYRrzNsxDOXRyN5A1pyWqZpc0DImg1g1U1f-2tYYgXDaaazfe0cYKq1kJJAmM7y4ECC07rcJzwCPjDr7wgYmF4QRuAWkusqBeyi3-GMeCy79CWHNMW-JOUZg560D5TPBJVEbQtmWs_KfoPFhKe8hP7DZOjvTPGlPHBJXWLH-wl7FD5iQVrmMb_2TgLULmgl0JpgBdVqLXkDoVpYqRIiOxBfY6ds3rEGV8leA2VQXU2XfnbAhYfY3Q_bH7FsBKNsxpRXTKkIMB85gKHu1cO42Knr5inhLZmtmU0mxqaZhpv9h_d0Mwv01Uxh4qXxjVMpRgto96F_-C231sbV2HqTWuB3a9pzUZ8DGaZLVqVGQmdYQ_razRteqJCc39gNZOX-hKa-GASDgEuhOil6N3cC46tFjL4osKq7G5QKTm9mik-lXDzY5NWo5dDjtUFoiYmtYIqE0yKut5G5Ptrp87Qfzb6hlb7JLxBdeentnZAnxE4dRFKThijWJrsmYD5MAfJWm8FGnfTMKwchyQYiKJGTgeVp36M", "e": "AQAB", "kty": "RSA", "kid": "a0"}, {"use": "sig", "n": "3c2XFHeXcUuJq-QAgC5dFfDM4zLt47E9GKGzH6hTed88XmbJvuAaaRENcv99-9EUINUvcT0xk-D8nK-HCtUelzbRgQG1TkkMy2hKiFhW6EmTptyGCSQkdxP-skVavZ-1B7I0L3GJo9FmsimKsrYRrzNsxDOXRyN5A1pyWqZpc0DImg1g1U1f-2tYYgXDaaazfe0cYKq1kJJAmM7y4ECC07rcJzwCPjDr7wgYmF4QRuAWkusqBeyi3-GMeCy79CWHNMW-JOUZg560D5TPBJVEbQtmWs_KfoPFhKe8hP7DZOjvTPGlPHBJXWLH-wl7FD5iQVrmMb_2TgLULmgl0JpgBdVqLXkDoVpYqRIiOxBfY6ds3rEGV8leA2VQXU2XfnbAhYfY3Q_bH7FsBKNsxpRXTKkIMB85gKHu1cO42Knr5inhLZmtmU0mxqaZhpv9h_d0Mwv01Uxh4qXxjVMpRgto96F_-C231sbV2HqTWuB3a9pzUZ8DGaZLVqVGQmdYQ_razRteqJCc39gNZOX-hKa-GASDgEuhOil6N3cC46tFjL4osKq7G5QKTm9mik-lXDzY5NWo5dDjtUFoiYmtYIqE0yKut5G5Ptrp87Qfzb6hlb7JLxBdeentnZAnxE4dRFKThijWJrsmYD5MAfJWm8FGnfTMKwchyQYiKJGTgeVp36M", "e": "AQAB", "kty": "RSA", "kid": "a1"}, {"use": "sig", "crv": "P-256", "kty": "EC", "y": "yDs4FrAZNnCuKZH07waOba9qrvbFauf_uD92m_B-7rM", "x": "NKRj4la2bcYIw9n6afKKmu80sskRuAehn0I0225QEDs", "kid": "a2"}, {"use": "enc", "crv": "P-256", "kty": "EC", "y": "4R0c33kuT72yom4dMFlArIbaa1ZbNt9EmVjNsOSgHhE", "x": "wpvaWGaH_Kc1bHUkravXisOpNHS6QFO_8jK8YTwEI_w", "kid": "a3"}]}
```

Comment:
Is this file supposed to be included in the commit? It seems to keep changing when I run it locally also.